components/plurals/src/lib.rs

```diff
@@ -878,12 +878,22 @@ where
 }
 
 #[derive(Debug, Clone, PartialEq, Eq)]
+#[cfg_attr(
+    feature = "serde",
+    derive(serde::Deserialize, serde::Serialize),
+    serde(transparent)
+)]
 /// A bag of values for different plural cases.
+///
+/// # Serialization
+///
+/// Although this type implements Serde traits, ICU4X developers and clients
+/// seeking a more efficient bitpacked representation should consider
+/// [`crate::provider::PluralElementsPackedCow`].
 pub struct PluralElements<T>(PluralElementsInner<T>);
```

Comment:
@robertbastian I added you to the review primarily to give you an opportunity to raise an objection to this change, in `icu_plurals`. Given the lack of comment in this file, is it safe to assume that you are okay with this change?

Reply:
Thanks for pointing this out, I do have objections. The serde impls on `PluralElementsInner` are currently only used in the human readable code path; this change uses them for all serializers through `PluralElements`. `skip_serializing_if` does not work for non-self-describing formats like postcard.